src/components/WalletFace/WalletFace.js

```diff
@@ -38,7 +38,7 @@ const WalletFace = ({
               iconName='dots'
               iconSize='medium'
               iconColor='white'
-              hasNotBorder
+              isBordered={false}
```

Comment:
boolean props should be false by default to avoid `isBordered={false}`
like any other html boolean attrs.

Reply:
Then I need to specify hasNotBorder or isNotBordered. Else the logic is broken.